tests/test_visualizer.py

```diff
@@ -0,0 +1,52 @@
+""" Implement tests for Visualizer class"""
+import pandas as pd
+import pytest
+import seaborn as sns
+
+from ftpvl.evaluation import Evaluation
+from ftpvl.styles import ColorMapStyle
+from ftpvl.visualizers import DebugVisualizer, SingleTableVisualizer
+
+
+class TestVisualizer():
+    """
+    Testing by partition.
+
+    DebugVisualizer()
+
+    SingleTableVisualizer()
+    """
+    def test_debugvisualizer(self):
```

Comment:
This is a good start. Let's discuss more robust testing of visualizations in the near future. It's literally one of the biggest problems I deal with, and there are various competing ways to do this.

Reply:
(this isn't a blocker, I'm resolving this for now)